samples/kubernetes/end2end/docs/03-wls-operator.md

```diff
@@ -82,12 +46,6 @@ kubectl get crd domains.weblogic.oracle
 NAME                      CREATED AT
 domains.weblogic.oracle   2019-05-28T07:17:26Z
 ```
-Verify that the domain CRD version is correct.
-```
-kubectl get crd domains.weblogic.oracle -o jsonpath="{.spec.version}"
-```
-The expected version should be `v5`. Because the operator is backward compatible, a domain resource with any version older than `v5` should be supported.
-
-Now the WebLogic Server Kubernetes Operator is running and it's monitoring the default namespace. Later we'll deploy a domain resource to the default namespace and the operator will be responsible for creating, running, and managing the WebLogic domain.
+Now the WebLogic Server Kubernetes Operator is running and it's monitoring all namespaces created  with label weblogic weblogic-operator=enabled.The operator will be responsible for creating, running, and managing any WebLogic domains, deployed to namespaces with such labels.
```

Comment:
with label weblogic weblogic-operator=enabled. ->  with the label `weblogic weblogic-operator=enabled`.
domains, deployed to namespaces with such labels. -> domains (no comma) deployed to namespaces with this label.

Reply:
done